src/main/plugin/iso19115-3.2018/schematron/schematron-rules-inspire.sch

```diff
@@ -609,23 +609,20 @@
 			<sch:report test="$otherConstraintInfo!='' and not($accessConstraints) and not($otherConstraints)" diagnostics="rule.constraints-allConstraints-success-en rule.constraints-allConstraints-success-fr"/>
 		</sch:rule>
 		
-		<sch:rule context="//mri:MD_DataIdentification/mri:resourceConstraints/*|
-			//*[@gco:isoType='mri:MD_DataIdentification']/mri:resourceConstraints/*|
-			//srv:SV_ServiceIdentification/mri:resourceConstraints/*|
-			//*[@gco:isoType='srv:SV_ServiceIdentification']/mri:resourceConstraints/*">
-			<sch:let name="accessConstraints" value="string-join(mco:accessConstraints/*/@codeListValue, ', ')"/>
-			<sch:let name="classification" value="string-join(mco:classification/*/@codeListValue, ', ')"/>
-			<sch:let name="otherConstraints" value="mco:otherConstraints/gco:CharacterString/text()"/>
-			<!-- <sch:let name="useLimitation" value="mco:useLimitation/*/text()"/> -->
-			<sch:let name="useLimitation" value="//mdb:identificationInfo/mri:MD_DataIdentification/mri:resourceConstraints/mco:MD_LegalConstraints/mco:useLimitation/gco:CharacterString/text()"/>
-			<!-- <sch:let name="useLimitation_count" value="count(mco:useLimitation/*/text())"/> -->
-			<sch:let name="useLimitation_count" value="count(//mri:resourceConstraints/*/mco:useLimitation/gco:CharacterString/text())"/>
-			<sch:report test="$accessConstraints!=''" diagnostics="rule.constraints-mco-accessConstraints-success-en rule.constraints-mco-accessConstraints-success-fr"/>
-			<sch:report test="$classification!=''" diagnostics="rule.constraints-classification-success-en rule.constraints-classification-success-fr"/>
-			<sch:assert test="$useLimitation_count" diagnostics="rule.constraints-useLimitation-failure-en rule.constraints-useLimitation-failure-fr"/>
-			<sch:report test="$useLimitation_count" diagnostics="rule.constraints-useLimitation-success-en rule.constraints-useLimitation-success-fr"/>
-		</sch:rule>
-
+		 <sch:rule context="//mdb:identificationInfo/* |
+                        //*[@gco:isoType='mdb:identificationInfo']/*|
+                        //srv:SV_ServiceIdentification/*|
+                        //*[@gco:isoType='srv:SV_ServiceIdentification']/*">
+                        <sch:let name="accessConstraints" value="string-join(mco:accessConstraints/*/@codeListValue, ', ')"/>
+                        <sch:let name="classification" value="string-join(mco:classification/*/@codeListValue, ', ')"/>
+                        <sch:let name="otherConstraints" value="mco:otherConstraints/gco:CharacterString/text()"/>
+                        <sch:let name="useLimitation_count" value="count(//mri:resourceConstraints/*/mco:useLimitation/gco:CharacterString/text())"/>
```

Comment:
I have still one doubt: 
if I wrote 
`<sch:let name="useLimitation_count" value="count(/mri:resourceConstraints/*/mco:useLimitation/gco:CharacterString/text())"/>`
then I got an error and the metadata is not fully validated. Could you please check this?